.github/workflows/ci.yml

```diff
@@ -87,3 +96,33 @@ jobs:
       - name: Run Unit Tests
         run: make testrun
 
+  # Create Release
```

Comment:
i recommend moving this job to its own workflow file named `release.yml`, with just `on:push:tags' as the trigger event. Since triggers are per workflow, as written this will run on every PR and every push to master as well as on every tag event.